### PR TITLE
Adds Teads publish script

### PR DESCRIPTION
**Pull request by arvy**

### CODE REVIEW

File: publish.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -xe
+
+# Install local teads-central
+docker run --rm -i docker-registry.teads.net/curl:master -sL http://ebzpub.ebuzzing.com/files/teads-central/get.sh | sh - 
+
+IMAGE=$(./teads-central vars image)
+HASH=$(./teads-central vars hash)
```
Comment (bjaglin) [inline publish.sh]: not used

File: publish.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+VERSION=$(docker run $IMAGE:$HASH version | tail -1 | sed s/" "//g | sed s/v//g)
```
Comment (bjaglin) [inline publish.sh]: not used (yet) ?

File: publish.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+docker build -t ${IMAGE}:9.6-alpine 9.6/alpine
```
Comment (bjaglin) [inline publish.sh]: I would go for dropping the alpine suffix in our own registry to be consistent with other images, just using version as tag

File: publish.sh (hunk 1 above)
Comment (arvy) [reply publish.sh]: this should be used in the build command

Comment (bjaglin) [reply publish.sh]: that's only if the `test.sh` tags it with HASH - have a look at other small repos for the usual test.sh/publish.sh

**Comment by arvy** [comment]
I'll add an empty test.sh to make sure it doesn't fail: https://github.com/ebuzzing/jenkins-job-definitions/pull/842

**Comment by arvy** [comment]
Found an easy way to test. ..so added that to `test.sh`. ..also moved the building of the image from `publish.sh` to `test.sh`

**Review by bjaglin — APPROVED** [review]
@yazgoo  is the version made available in that PR (9.6) in line with what we will have in the prod deployment of airflow?

**Comment by yazgoo** [comment]
We're using latest, which incidentally is 9.6 .
I will pin it to 9.6 though.

**Pull request merged**